cmd/archeio/internal/app/buckets_integration_test.go

```diff
@@ -20,10 +20,18 @@ limitations under the License.
 package app
 
 import (
+	"bytes"
+	"crypto/sha256"
+	"encoding/hex"
+	"io"
+	"net/http"
 	"testing"
+
+	"k8s.io/registry.k8s.io/pkg/net/cloudcidrs"
 )
 
-func TestCachedBlobChecker(t *testing.T) {
+func TestIntegrationCachedBlobChecker(t *testing.T) {
```

Comment:
integration tests should be prefixed `TestIntegration` for running `make integration`